packages/xstate-wallet/src/ui/enable-ethereum-workflow.tsx

```diff
@@ -34,6 +121,16 @@ export const EnableEthereum = (props: Props) => {
     }
   };
 
+  const connectionBanner = (
+    <ConnectionBanner currentNetwork={currentNetwork} requiredNetwork={targetNetwork}>
+      {{
+        notWeb3CapableBrowserMessage: <NotWeb3Browser />,
+        noNetworkAvailableMessage: <NoNetwork />,
+        onWrongNetworkMessage: <WrongNetwork />
+      }}
+    </ConnectionBanner>
```

Comment:
Am I right in thinking this is pretty close to just using the default values? Is the main advantage that you get to resize the banner by doing this?